### PR TITLE
BAH Stressful Incident Description Page

### PR DESCRIPTION
**Pull request by robfelder**
## Description
This PR covers the Stressful Incident description page of the 781 flow.

## Testing done
Manual testing
Unit test written

## Screenshots
screen shot 2018-10-03 at 10 37 31 am

## Acceptance criteria
- [x] The user should be presented with guidance text per the Content Document
- [x] The user is able to input the incident description via free form text 
- [x] Special characters should be allowed
- [x] There is no limit on how much text the user can enter in the Description of the incident text box
- [ ] The PTSD type that the user previously selected (either Personal Assault or Military Sexual Trauma) should be populated as the _$PTSDclassification_ on this page, per the Content Document
- [ ] If the user selected **more than one** PTSD Type for 0781a (Personal assault and Military sexual trauma) then the following is expected:
                - then 'Personal assault & Military sexual trauma' replaces $PTSDclassification in header of 
                781 workflow
               - then 'Personal assault and Military sexual trauma' replaces $PTSDclassification in body of 
                781 workflow

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by robfelder** [comment]
@imeecuison @aaronjlech, when you get a moment, could you review please.

**Note:** I still need to implement the PTSD Classification though.

File: src/applications/disability-benefits/781/helpers.js [hunk 1]
```diff
@@ -108,3 +108,9 @@ export const documentDescription = () => (
     </p>
   </div>
 );
+
+export const stressfullIncidentDescriptionTitle = () => (
```
Comment (aaronjlech) [inline src/applications/disability-benefits/781/helpers.js]: `stressfull` should be `stressful`

File: src/applications/disability-benefits/781/pages/stressfulIncSecDesc.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { ptsdNameTitle, stressfullIncidentDescriptionTitle } from '../helpers';
```
Comment (aaronjlech) [inline src/applications/disability-benefits/781/pages/stressfulIncSecDesc.js]: +1

File: src/applications/disability-benefits/781/pages/stressfulIncSecDesc.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+export const uiSchema = {
+  'ui:title': ptsdNameTitle,
+  'ui:description': stressfullIncidentDescriptionTitle,
```
Comment (aaronjlech) [inline src/applications/disability-benefits/781/pages/stressfulIncSecDesc.js]: +1

**Review by aaronjlech — CHANGES_REQUESTED** [review]
A couple of spots stressful has an extra L, other than that looks good.

**Review by imeecuison — APPROVED** [review]
Awesome!

**Comment by robfelder** [comment]
@aaronjlech spelling mistakes updated. Thanks for pointing that out.

**Review by aaronjlech — APPROVED** [review]
@robfelder LGTM!!

**Pull request merged**